src/couch/src/couch_util.erl

```diff
@@ -792,3 +799,31 @@ version_to_binary(Ver) when is_list(Ver) ->
     Ver1 = lists:reverse(lists:dropwhile(IsZero, lists:reverse(Ver))),
     Ver2 = [erlang:integer_to_list(N) || N <- Ver1],
     ?l2b(lists:join(".", Ver2)).
+
+-compile({inline, [hex/1]}).
+
+%% erlfmt-ignore
+hex(X) ->
+    % Table of all pairs of hex characters for a byte:
```

Comment:
Might add a note that we're encoding the hex as ascii directly here. Took me a few minutes to realize what was going on.